Software/Development/SemesterProject/App/closedLoopControlManager.c

```diff
@@ -0,0 +1,68 @@
+/*
+ * closedLoopControlManager.c
+ *
+ *  Created on: 25. apr. 2019
+ *      Author: Nicolai Fransen
+ */
+
+#include "closedLoopControlManager.h"
+
+void runClosedLoopControl(void)
+{
+    float movementReference = 0;
+    float iqReference = 0, idReference = 0;
+    float abcCurrents[2];
+    float theta = 0;
+    volatile dqObject dqCurrents, dqVoltages;
+    alphaBetaObject abVoltages;
+
+    movementReference = getMovementReference();         // Reads torque or speed reference

```

Comment:
Lines 19 to 22 are definitely ok but I don't know whether you could maybe make things clearer, like a better story explanation?? Not big deal, you can neglect the comment.

Reply:
¿Do you mean better comments or better names? Or both?